### PR TITLE
Basic sass principles applied to stylings and general tidy up

### PR DESCRIPTION
**Pull request by tanyapowell**
@empowerhack/drawmylife 
- font awesome gem added (Issue #60) 
- all colours assigned to variables
- began assigning commonly used dimensions to variables
- each page/view/component in its own modular file

Will continue with #58 as I go along

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -14,4 +14,4 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
-//= require bootstrap-sprockets
\ No newline at end of file
+= require bootstrap-sprockets
```
Comment (steverabino) [inline app/assets/javascripts/application.js]: I don't think this change is actually needed.

It's absolutely absurd, I know, but `//=` is a directive, not a commented out line, and I think removing the `//` may have some unwanted effects

Comment (steverabino) [inline app/assets/javascripts/application.js]: I can't find an easy how-to guide which shows this, but you'll see examples dotted around on http://guides.rubyonrails.org/asset_pipeline.html

**Comment by steverabino** [comment]
The rest of the changes I'll need to check out the branch and have a look at when I go home - will have thumbs/comments tonight though

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (tanyapowell) [inline app/assets/javascripts/application.js]: Oh ok! When I was running the app locally I was getting an error saying that `bootstrap-sprockets` was missing. I'll put it back to how it was and re-push

**Comment by steverabino** [comment]
LGTM :+1:

**Pull request merged**